client/my-sites/domains/components/domain-and-plan-package/navigation/index.jsx

```diff
@@ -46,6 +55,14 @@ export default function DomainAndPlanPackageNavigation( props ) {
 				<li>{ translate( 'Complete your purchase' ) }</li>
 			</ol>
 			<div className="domain-and-plan-package-navigation__step-indication">{ stepIndication }</div>
+			{ props.showSkipPlans && (
+				<div className="domain-and-plan-package-navigation__skip">
+					<Button borderless="true" onClick={ goSkipPlan }>
+						<span>{ translate( 'Skip' ) }</span>
+						<Gridicon icon="chevron-right" />
+					</Button>
+				</div>
+			) }
```

Comment:
Should we show the "Skip >" button on the Plans page for the "Search for a domain" flow too?

Reply:
I updated the code, the "skip" button is now a copy, and we show this only for the My Home Upsell flow.
Btw, the domain page shows a `decide later` that will be removed for my home upsell flow.